deploy/keys/keyvault-update.sh

```diff
@@ -5,6 +5,9 @@
 # -----------
 # Service Bus auth rule names are hardcoded
 
+# should not be used as limited by only supporting the primary keys
+exit -1
```

Comment:
This should probably also print a warning to stderr

Reply:
done